components/brave_new_tab_ui/brave_new_tab_searchbox.mojom

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+module brave_new_tab_searchbox.mojom;
```

Comment:
how about we centralize this to a small component at `components/brave_search` and give it a spiffy name, like `brave_search.mojom.SearchHandler`, instead of tying to Private NTP

Reply:
i have added a seperate mojom for private tab, ptal